scripts/generate_duplicate_sheet.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+import csv
+import itertools
+from datetime import datetime, timedelta
+
+from bugbug import bugzilla
+from bugbug.models.duplicate import DuplicateModel
+
+m = DuplicateModel()
+m.train()
+
+
+REPORTERS_TO_IGNORE = {"intermittent-bug-filer@mozilla.bugs", "wptsync@mozilla.bugs"}
+delta = timedelta(days=21)
+test_bugs = bugzilla.download_bugs_between(datetime.now() - delta, datetime.now())
```

Comment:
https://github.com/mozilla/bugbug/pull/548#discussion_r290682355 still kind of applies.
This will add the bugs to the DB, so when you run the script again you will train the model on these bugs too.